pages/linux/fstab.md

```diff
@@ -0,0 +1,42 @@
+
+# fstab
+
+>/etc/fstab automatically mounts partitions with different options.
+>It is not a command, but a text file you edit with an editor of choice.
+>More information: <https://manned.org/fstab>.
+>DO NOT edit this file if you don't know what you're doing.
+
+- The file is seperated into different sections for each partition you want to auto-mount:
+
+>UUID/Label | Directory to mount to | Type of the partition | Options for mounting | Dump | Order in which partitions are read by 'fsck'
```

Comment:
```suggestion
`UUID/Label | Directory to mount to | Type of the partition | Options for mounting | Dump | Order in which partitions are read by 'fsck'`
```

Reply:
The first is the code description and the second is always the code example. Don't reverse the order.